shared/js/utils.js

```diff
@@ -36,6 +36,12 @@ require.scopes.utils = ( () => {
         }
     }
 
+    function capitalizeFirstLetter (string) {
+        if (typeof string !== 'string') { return '' }
+
+        return string[0].toUpperCase() + string.substring(1)
+    }
+
```

Comment:
There's a way better way to do this -- am using npm `titleize` in the `site` template. Please use that.

Reply:
Titleize does every word though?

Reply:
Maybe we can just switch to an all-in-one solution, e.g. https://www.npmjs.com/package/change-case